README.md

```diff
@@ -0,0 +1,181 @@
+<a name="readme-top">HedgeHog</a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="./app/assets/images/app.JPG" alt="app-home" width="300"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>HedgeHog</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 HedgeHog- Budget ](#-hedgehog--budget-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- INTRO -->
+# 📖 HedgeHog- Budget <a name="about-project"></a>
+
+> HedgeHog is a Budget app built with Ruby on Rails. You can create categories and have a list of transactions associated with a category, you can add new transactions and see the money you spent in each category and transaction.
+
+## 🛠 Built With <a name="built-with"></a>
+1. Ruby
+2. Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - List Categories
+> - Add a new Category
+> - List Transactions
+> - Add a new Transaction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+## 🚀 Live Demo <a name="live-demo"></a> 
+
+
+
+- [Live Demo Link](https://hedgehog.onrender.com/)
+- [Presentation Video](https://www.loom.com/share/6110bddc253c4c4fbd6d39d203b2c46f)
```

Comment:
- Good job on your video presentation and taking your users through on how to use your project, but according to the project requirements, your project should contain a highlight about a piece of the code the you find interesting or proud of making